eb_hooks.py

```diff
@@ -54,13 +54,25 @@ def get_rpath_override_dirs(software_name):
 
     return rpath_injection_dirs
 
+def set_Pillow_envvars(ec):
```

Comment:
I would suspect that there are more cases where this would be necessary. It might be better to make a generic method and move the `if ec.name == 'Pillow'` into the appropriate hook and then just call the generic method.

Reply:
A better name might be `add_compat_layer_to_cpath` and `add_compat_layer_to_library_path`. I would separate the two as I'd be worried that setting `LIBRARY_PATH` will have knock-on effects

Reply:
Again, see also https://gitlab.com/eessi/support/-/issues/9, where we sort of agreed that the best way forward for this one is a custom easyblock for Pillow that does what's necessary.

We haven't seen any other cases that warrant an approach like this.

If that pops up, we could consider lifting whatever is necessary into framework so you can easily opt-in to it somehow.

Reply:
I actually have seen it before, and I finally found the issue https://github.com/EESSI/software-layer/pull/103